kpi/renderers.py

```diff
@@ -122,6 +124,54 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
         )
 
 
+class SubmissionRendererExportBase(renderers.BaseRenderer, ExportBase):
+    def get_export_object(
+        self, renderer_context: Dict
+    ) -> Tuple[formpack.reporting.Export, Generator]:
+        view = renderer_context['view']
+        self.user = view.request.user
```

Comment:
Do you think it would be useful to add a comment here about why `self.user` needs to be set?

Reply:
Good idea 👍